<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestEventsDlgClass</class>
 <widget class="QDialog" name="RequestEventsDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download Events</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>332</width>
        <height>528</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageUsers">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>332</width>
            <height>416</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Users</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableView" name="tvUsers">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageEvents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>119</width>
            <height>116</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Events</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="checkSave">
             <property name="text">
              <string>SAVE</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkMode">
             <property name="text">
              <string>MODE</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkSCM">
             <property name="text">
              <string>SCM_COMMIT</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkChat">
             <property name="text">
              <string>CHAT</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>205</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageTimeSpan">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>152</width>
            <height>92</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Time span</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QDateTimeEdit" name="dtStart">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="displayFormat">
              <string>M/d/yyyy hh:mm</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>to</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="dtEnd">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="displayFormat">
              <string>M/d/yyyy hh:mm</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>229</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pagePhases">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>332</width>
            <height>416</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Phases</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QCheckBox" name="checkProject">
             <property name="text">
              <string>Project setting up</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkPrototyping">
             <property name="text">
              <string>Prototyping</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkCoding">
             <property name="text">
              <string>Coding</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkTesting">
             <property name="text">
              <string>Testing</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkDeployment">
             <property name="text">
              <string>Deployment</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelFuzziness">
             <property name="text">
              <string>Division fuzziness</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sliderFuzziness">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="pageStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
             <property name="tracking">
              <bool>true</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>131</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RequestEventsDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>-24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RequestEventsDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>-20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
